Makefile

```diff
@@ -25,6 +25,15 @@ bench-simple: $(benchcmp)
 	@go test -v -run '^$$' -bench /codeResponse -benchmem -benchtime 3s -cpu 1 ./json | tee segmentio-encoding-json.txt
 	benchcmp encoding-json.txt segmentio-encoding-json.txt
 
+bench-master: $(benchcmp)
+	git stash
+	git checkout master
+	@go test -v -run '^$$' -bench /codeResponse -benchmem -benchtime 3s -cpu 1 ./json | tee segmentio-encoding-json-master.txt
+	git checkout -
+	git stash pop
+	@go test -v -run '^$$' -bench /codeResponse -benchmem -benchtime 3s -cpu 1 ./json | tee segmentio-encoding-json.txt
+	benchcmp segmentio-encoding-json-master.txt segmentio-encoding-json.txt
```

Comment:
What do you think about moving these files to `/tmp`?

Reply:
No preference. Just copied `bench-simple`'s behavior. Are you thinking we should change both?

Reply:
Maybe, I find it annoying to have files left around in the repository when running tests, but it's very much personal taste :)